core/kazoo_number_manager/src/carriers/knm_bandwidth2.erl

```diff
@@ -54,33 +55,48 @@
 
 -ifdef(TEST).
 -define(BW2_ACCOUNT_ID, "eunit_testing_account").
+-define(IS_SANDBOX_PROVISIONING_TRUE, 'true').
```

Comment:
If you don't know already: calls to kapps_config and the like return the Default argument when running tests.